modules/spo-binding-workloads.adoc

```diff
@@ -35,10 +35,16 @@ spec:
   profileRef:
     kind: {kind} <1>
     name: profile <2>
-  image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
+  image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21 <3>
 ----
 <1> The `kind:` variable refers to the name of the profile.
 <2> The `name:` variable refers to the name of the profile.
+<3> You can enable a default security profile by using a wildcard in the image attribute: `image: "*"`
++
+[IMPORTANT]
+====
+Using the `image: "*"` wildcard attribute binds all pods with a default security profile in a given namespace.
```

Comment:
@Vincent056 I am unsure about two points. Could you please help to double confirm? Thanks.
Firstly, I tested this function. Actually, it only works for newly created pods(it doesn't work for the existing pods before profilebinding created). Maybe we can say 
Using the `image: "*"` wildcard attribute binds all new pods with a default security profile in a given namespace.

Secondly, there is a Import severity bug when trying to delete the profilebinding: https://issues.redhat.com/browse/OCPBUGS-37557. Do we need to add it to doc? Thanks.

Reply:
> @Vincent056 I am unsure about two points. Could you please help to double confirm? Thanks. Firstly, I tested this function. Actually, it only works for newly created pods(it doesn't work for the existing pods before profilebinding created). Maybe we can say Using the `image: "*"` wildcard attribute binds all new pods with a default security profile in a given namespace.
> 
> Secondly, there is a Import severity bug when trying to delete the profilebinding: https://issues.redhat.com/browse/OCPBUGS-37557. Do we need to add it to doc? Thanks.

that's correct, it will not work for the old pods.

for second, have you removed all the pods that uses the binding before you remove the profile @xiaojiey

Reply:
I addressed @xiaojiey's feedback, updating the text in the callouts and changing the note to "binds new pods...". 

Anything else I can add? @Vincent056 @BhargaviGudi